dataline-commons/src/main/resources/log4j2.xml

```diff
@@ -1,32 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="INFO">
+    <Properties>
+        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %X - %m%n</Property>
+    </Properties>
 
   <Appenders>
-    <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n"/>
+    <Console name="Default" target="SYSTEM_OUT">
+      <PatternLayout pattern="${default-pattern}"/>
     </Console>
+    <Routing name="Context">
+        <Routes pattern="${ctx:context}">
+            <Route key="worker">
+                <File name="${ctx:context}-${ctx:job_id}" fileName="${ctx:job_root}/logs.log">
+                    <PatternLayout pattern="${default-pattern}"/>
+                </File>
+            </Route>
+        </Routes>
+        <IdlePurgePolicy timeToLive="15" timeUnit="minutes"/>
```

Comment:
15 mins to live seems a little short, maybe a few days initially? (I'm imagining if someone filed a bug report, we'd want them to be able to pull logs)

Reply:
This is not meant to remove the logs but the log4j job dedicated appender from the memory.